internal/saucecloud/cloud.go

```diff
@@ -885,3 +889,21 @@ func (r *CloudRunner) getAvailableVersionsMessage(frameworkName string) string {
 	m += "\n"
 	return m
 }
+
+func (r *CloudRunner) getTestHistory() (insights.TestHistory, error) {
+	client := insights.Client{
+		HTTPClient:  &http.Client{},
```

Comment:
1. Never use the default client without setting the timeout, as the timeout is infinite.
2. Why break the established pattern we have in saucectl where the dependencies are passed down? 🤔